pages/linux/mt.md

```diff
@@ -0,0 +1,32 @@
+# mt
+
+> Control magnetic tape drive operation (commonly LTO tape).
+> More information: <https://manned.org/mt>.
+
+- Check tape status:
+
+`mt -f {{/dev/[nst|st]N}} status`
```

Comment:
We don't allow `[]` only this:
```suggestion
`mt -f /dev/{{nst|st}}N status`
```

Reply:
This is super confusing, who would know what that means? [|] is usually used for an either/or set.
Why not simply leave it at `<device>`, everyone will know what is meant and the **tl;dr** doesn't have to be so cluttered.

Reply:
Let's wait for @sbrl (check this :thread: https://github.com/tldr-pages/tldr/pull/7273#discussion_r737981287)

Reply:
I suggest picking the most common one and sticking with it. Ref https://www.linuxquestions.org/questions/linux-hardware-18/difference-between-dev-st0-and-dev-nst0-493760/ I suggest `/dev/nst`:
```suggestion
`mt -f {{/dev/nstX}} status`
```
Also, are there any long forms of these arguments at all?

Reply:
I do not know any long forms. Many tape related commands are actually lacking a lot of really good documentation and tips, that's why I think they're a great addition for this project.
`nst` is indeed the most common one, so let's go with that.